### ISSUE TITLE
[$40][MSFT-212] User can't focus on the the 'Sort by' using tab

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2874]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges?bucket=my
2. Press tab key till the 'Sort by' are selected

**Expected Result**
User should be able to select the 'Sort by' using tab

**Actual Result**
User can't focus on the 'Sort by' using tab

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F21

**Comment by ThomasKranitsas** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095964 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095964 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095964 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/2930

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2874 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2874 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2874 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2874 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2874 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097090 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097090 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by anonymousjaggu** [comment]
@PrakashDurlabhji Please check the comment in `MR`

**Comment by PrakashDurlabhji** [comment]
@veshu failed to solve it hence releasing

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097090 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by afrisalyp** [comment]
The feature uses react-select. https://www.npmjs.com/package/react-select
Need changes on the module.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097090 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu can you please increase the prize for this ticket? It took a lot of debugging to solve the issue. Thanks

**Comment by veshu** [comment]
@r0hit-gupta sure.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097090 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu thanks a lot :)

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by ajefts** [comment]
Verified in prod:

![image](https://user-images.githubusercontent.com/3812270/62155359-a1465f80-b2d6-11e9-8c51-cda39da6404b.png)

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097090<br/><br/>```This is an automated message for crazyk via Topcoder X```
